selftests/functional/test_basic.py

```diff
@@ -646,6 +646,9 @@ def test_simpletest_fail(self):
                          "Avocado did not return rc %d:\n%s" %
                          (expected_rc, result))
 
+    @unittest.skipIf(os.environ.get("AVOCADO_CHECK_FULL") != "1",
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
Sure, I don't actually think this test should be here as it's basically a performance/stress test.